src/libcephfs.cc

```diff
@@ -78,13 +78,11 @@ struct ceph_mount_info
   int init()
   {
     common_init_finish(cct);
+	  
     int ret;
-    struct client_callback_args args = [this]{
-        struct client_callback_args args;
-        args.handle = this;
-        args.umask_cb = umask_cb;
-        return args;
-    }();
+    struct client_callback_args args = {};
```

Comment:
again, please define `args` where is used. also, might want to drop `struct`, it's required by C, not C++.

Reply:
If I declare `client_callback_args = {}` on line 116 It is giving me the error
`transfer of control bypasses initialization of: -- variable "args"`. so do I just have to initialize `args` on line 116?

Reply:
I think where it's defined is okay for the reason you stated. You can drop the "struct" though.